db/seeds.rb

```diff
@@ -3,3 +3,4 @@
 require_relative 'seeds/jokes.rb'
 require_relative 'seeds/users.rb'
 require_relative 'seeds/orders.rb'
+require_relative 'seeds/order_item.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.